extensions/src/main/scala/ai/h2o/sparkling/extensions/serde/ChunkAutoBufferWriter.scala

```diff
@@ -17,50 +17,48 @@
 
 package ai.h2o.sparkling.extensions.serde
 
-import SerializationUtils._
 import java.io.{Closeable, OutputStream}
+import java.sql.Timestamp
 import java.util.UUID
 
 import water.fvec.{ChunkUtils, Frame}
 import water.parser.BufferedString
 import water.{AutoBuffer, DKV}
 
-final class ChunkAutoBufferWriter(val outputStream: OutputStream) extends Closeable {
+final class ChunkAutoBufferWriter(val outputStream: OutputStream) extends Closeable with ChunkSerdeConstants {
 
   private val buffer = new AutoBuffer(outputStream, false)
 
   def writeChunk(frameName: String, chunkId: Int, expectedTypes: Array[Byte], selectedColumnIndices: Array[Int]): Unit = {
     val frame = DKV.getGet[Frame](frameName)
     val chunks = ChunkUtils.getChunks(frame, chunkId)
     val numberOfRows = chunks(0)._len
-    sendInt(buffer, numberOfRows)
+    writeInt(numberOfRows)
```

Comment:
I think that in the future we can remove this from the protocol as we are streaming the data now, but I wouldn't do it as part of this PR as that would be bigger change